### PR TITLE
Added CPU Manager content to 4.0

### PR DESCRIPTION
**Pull request by ahardin-rh**
Preview Build: http://file.rdu.redhat.com/~ahardin/04302019/using-cpu-manager-40/scalability_and_performance/using-cpu-manager.html

https://jira.coreos.com/browse/OSDOCS-41

Dev team review tracker: https://jira.coreos.com/browse/ATS-170

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@ravisantoshgudimetla @jeremyeder @derekwaynecarr PTAL and let me know that this content is all still accurate for 4.0. Thanks!

**Comment by ahardin-rh** [comment]
@jeremyeder PTAL

**Comment by jeremyeder** [comment]
No, these docs are incorrect as-is - the section on how to edit the kubelet configuration will be changing. @sjenning

**Comment by ahardin-rh** [comment]
@jeremyeder  Any updates here? Our docs team is looking to get all 4.0 docs completed for a total QE review by February 22.

**Comment by ahardin-rh** [comment]
@jeremyeder Any updates?

**Comment by ahardin-rh** [comment]
@zvonkok Any updates?

**Comment by ahardin-rh** [comment]
@zvonkok I applied the latest changes. PTAL.
@mffiedler Please provide a QE review

Latest preview build:
http://file.rdu.redhat.com/~ahardin/04302019/using-cpu-manager-40/scalability_and_performance/using-cpu-manager.html

File: modules/setting-up-cpu-manager.adoc [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+----
++
+This adds the CPU Manager feature to the `KubeletConfig` and, if needed, the
+Machine Config Operator (MCO) reboots the node. To enable CPU Manager, a reboot
+is not needed.
+
+. Check for the merged `KubeletConfig`:
++
+----
+# oc get machineconfig 99-worker-XXXXXX-XXXXX-XXXX-XXXXX-kubelet -o yaml | grep ownerReference -A7
```
Comment (zvonkok) [inline modules/setting-up-cpu-manager.adoc]:
```suggestion
# oc get machineconfig 99-worker-XXXXXX-XXXXX-XXXX-XXXXX-kubelet -o json | grep ownerReference -A7
```
This is definitely `json` format :)

**Comment by zvonkok** [comment]
@ahardin-rh lgtm

**Comment by mffiedler** [comment]
Looks good to me as well if @zvonkok 's requested change for -o json.

File: modules/setting-up-cpu-manager.adoc [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+. Verify that the `cgroups` are set up correctly. Get the process ID (PID) of the
+`pause` process:
++
+----
+# ├─init.scope
+│ └─1 /usr/lib/systemd/systemd --switched-root --system --deserialize 17
+└─kubepods.slice
+  ├─kubepods-pod69c01f8e_6b74_11e9_ac0f_0a2b62178a22.slice
+  │    ├─crio-b5437308f1a574c542bdf08563b865c0345c8f8c0b0a655612c.scope
+  │ │ └─32706 /pause
```
Comment (zvonkok) [inline modules/setting-up-cpu-manager.adoc]:
```suggestion
  │    └─32706 /pause
```
Missalignment, looks not right.

File: modules/setting-up-cpu-manager.adoc [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+
+. Verify that the `cgroups` are set up correctly. Get the process ID (PID) of the
+`pause` process:
++
+----
+# ├─init.scope
+│ └─1 /usr/lib/systemd/systemd --switched-root --system --deserialize 17
+└─kubepods.slice
+  ├─kubepods-pod69c01f8e_6b74_11e9_ac0f_0a2b62178a22.slice
+  │    ├─crio-b5437308f1a574c542bdf08563b865c0345c8f8c0b0a655612c.scope
```
Comment (zvonkok) [inline modules/setting-up-cpu-manager.adoc]:
```suggestion
  │ ├─crio-b5437308f1a574c542bdf08563b865c0345c8f8c0b0a655612c.scope
```

File: modules/setting-up-cpu-manager.adoc (hunk 2 above)
Comment (zvonkok) [inline modules/setting-up-cpu-manager.adoc]:
```suggestion
  │ └─32706 /pause
```

**Comment by ahardin-rh** [comment]
@zvonkok @mffiedler Thank you! I applied the suggested changes. Merging!

**Comment by mburke5678** [comment]
@jeremyeder @zvonkok @mffiedler Not sure where to ask. The _Setting up CPU Manager_ steps call for [creating a dynamic `KubeletConfg`](https://github.com/openshift/openshift-docs/pull/12827/files#diff-71807971cdded0a624ac8b40a35c96f3eb927d6ff36a3f0e2a5f545f18ae31b2R52).
I wanted to check if this is the same thing as [Kubernetes Dynamic Kubelet Configuration](https://kubernetes.io/blog/2018/07/11/dynamic-kubelet-configuration/), which is [slated for deprecation in Kube 1.22/OpenShift 4.9](https://issues.redhat.com/browse/OCPNODE-590). 
Does the step in _Setting up CPU Manager_ need to be updated in 4.9+?

**Pull request merged**